test/unit/assorted/server_discovery_and_monitoring.spec.test.ts

```diff
@@ -253,7 +256,10 @@ async function executeSDAMTest(testData) {
             error instanceof MongoServerError
           );
         };
-        expect(thrownError).to.satisfy(isApplicationError);
+        expect(
+          thrownError,
+          'expected the error thrown to be one of MongoNetworkError, MongoNetworkTimeoutError or MongoServerError. Referred to in the spec as an "Application Error"'
```

Comment:
```suggestion
          'expected the error thrown to be one of MongoNetworkError, MongoNetworkTimeoutError or MongoServerError (referred to in the spec as an "Application Error")'
```

Reply:
agreed, pulled this into the other fixes.